app/modals/stacking/stacking-modal.tsx

```diff
@@ -37,13 +48,22 @@ enum StackingModalStep {
   NetworkError,
 }
 
+const poxClient = new POX(DEFAULT_STACKS_NODE_URL);
```

Comment:
This needs to be pulled in from the `selectedNodeApi` selector, so it updates when user changes node.